File: 주다애/Sort/prob1.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int len = array.length;
        int cLen = commands.length;
        
        int[] answer = new int[cLen];
        for(int i = 0; i < cLen; i++) {
            int[] temp = Arrays.copyOfRange(array, commands[i][0] - 1, commands[i][1]);
            Arrays.sort(temp);
            answer[i] = temp[commands[i][2] - 1];
        }
        return answer;
    }
}

File: 전예진/Sorting/prob1.py
def solution(array, commands):
    answer = []
    
    lst = []
    for n in range(len(commands)):  
        i = commands[n][0]
        j = commands[n][1]
        k = commands[n][2]

        lst = array[i-1:j]
        lst.sort()
        answer.append(lst[k-1])

    return answer
